tests/spec/examples.py

```diff
@@ -743,25 +743,6 @@ def it_works(self, in_example_dir):
         )
 
 
-class DescribeStartMessageSuccess(object):
-
-    @pytest.yield_fixture
-    def service_name(self):
-        yield 'start-message'
-
-    def it_prints_a_start_message_on_successful_startup(self, in_example_dir):
-        assert_command(
-            ('pgctl', 'start', 'start-message'),
-            '',
-            '''\
-[pgctl] Starting: start-message
-[pgctl] Started: start-message
-Service has started at localhost:9001
-''',
-            0
-        )
```

Comment:
Please keep this test, just update the example to the new scheme.
This will prove out the theory that the old behavior can still be implemented with a simple update.